ACD-618 (#30)
Actindo-AG/Documentation: Channels/Integration/01a_Shopware6.md

> [Info] For the *Omni-Channel* module version 4.1.0 or higher, the *Shopware 6* plugin is required in at least version 4.0.0.

[comment]: <> (Stimmt Version? 4.0.0 ist die neueste. Stimmen prerequisites? Oder besser Zwei subprocedures: 1 create integration in Shopware 6 + create connection in Core1? -> Verweis auf Manage connections?)

#### Procedure

*Omni-Channel > Settings > Tab CONNECTIONS*

![Connection](../../Assets/Screenshots/Channels/Settings/Connections/Connection.png "[Connection]")

1. Click the [Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.    
  The *Create connection* view is displayed.

  ![Create connection](../../Assets/Screenshots/Channels/Settings/Connections/CreateConnection.png "[Create connection]")

2. Enter a name for the connection in the *Name* field.

3. Click the *Driver* drop-down list and select *Shopware 6* driver.  
  The *Credentials* section is displayed below the drop-down list.

4. Enter the shop URL in the *URL* field.  

> 5. In a new browser, go to your Shopware 6 account and log in. 

Author: ```suggestion
5. In a new browser window, go to your Shopware 6 account and log in. 
```

6. In the Shopware 6 backend, go to *Settings > System > Integrations* and click the [Create integration] button in the top right corner.  
  The *Create integration* window is displayed.

  ![Create integration](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/CreateIntegration.png "[Create integration]")

7. Enter a name for the integration in the *Name* field and enable the *Administrator* toggle.

> 8. Copy the code provided in the *Access key ID* field to your clipboard.

Author: ```suggestion
8. Copy the access key id in the *Access key ID* field to your clipboard.
```
alternativ identification number

> 9. Switch to the *Actindo Core1 Platform* and paste the copied access ID from your clipboard in the *Access key ID* field in the *Credentials* section.  

Author: ```suggestion
9. Switch to the *Actindo Core1 Platform* and paste the access key ID from your clipboard in the *Access key ID* field in the *Credentials* section.  
```

> 10. Repeat the steps **8** and **9** for the *Secret access key* field.

Author: ```suggestion
10. Repeat the steps **8** and **9** for the secret access key in the *Secret access key* field.
```

11. Switch to your Shopware 6 account and click the [Save integration] button.  
  The integration is saved. The *Success* pop-up window is displayed.

  ![Integration  created](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/IntegrationCreated.png "[Integration created]")

12. Switch to the *Actindo Core1 Platform* and click the [SAVE] button.  

>   The connection has been created.   
>   The *CONNECTIONS* tab in the *Settings* menu entry of the *Omni-Channel* module is displayed when the connection has been established. The *Shopware 6* connection is displayed in the list of connections.

Author: ```suggestion
  The connection has been created.  The *CONNECTIONS* tab in the *Settings* menu entry of the *Omni-Channel* module is displayed when the connection has been established. The *Shopware 6* connection is displayed in the list of connections.
```

12. If necessary, continue to [Configure the Shopware 6 connection](#configure-the-shopware-6-connection).

[comment]: <> (Was genau passiert nach SAVE in Core1? Einige Minuten Zeit/Sync/Platform initialized? Satz aus Shopify genommen.)

## Configure the Shopware 6 connection   

After the connection to a Shopware 6 shop has been established, further settings can be configured for the connection.

#### Prerequisites

A Shopware 6 connection has been established, see [Create a Shopware 6 connection](#create-a-shopware-6-connection).

#### Procedure

*Omni-Channel > Settings > Tab CONNECTIONS*

![Shopware 6 connection](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/Connection.png "[Shopware 6 connection]")

1. Click the Shopware 6 connection in the list of connections.   
  The *Edit connection* view is displayed. By default, the *Credentials* tab is selected.

  ![Edit connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/EditConnectionCredentials.png "[Edit connection credentials]")

2. Click the *Settings* tab.   
  The *Settings* tab is displayed. By default, the *Default language* setting is selected.

>   > [Info] In Shopware 6 there is a standard language, which is configured during the installation and cannot be subsequently edited. The standard language cannot be detected via API and must be therefore manually configured in the *Actindo Core1 Platform* after the connection has been established. When uploading products, Shopware 6 requires some fields to be filled for the standard language.

Author: Vielleicht sollte diese Info in den Einführungssatz mit rein? Lass uns drüber sprechen. 
Auf jeden Fall fehlt hier aber der Step, dass ich die Standardsprache einstelle...

 
  + If desired, enable for each payment method the toggles of all states for which the orders are to be imported. If you select no payment states for a certain payment method, all orders with this payment method, regardless their state, will be imported. All states available in Shopware 6 are displayed for each payment method. 
  The following payment methods are available:   
    - *Paypal*  
    - *Cash on delivery*  
    - *Prepayment*  
    - *Pay upon invoice*  
    - *Ratepay Prepayment*
    - *Ratepay Direct Debit*
    - *Ratepay Open invoice*
    - *Ratepay Payment by Installment*
    - *Ratepay 0% Financing*
    - *PAYONE Apple Pay*
    - *Credit card (Visa/Mastercard)*
    - *Payment SEPA Direct Debit*
    - *PAYONE PayPal*
    - *Mastercard*
    - *PAYONE Paysafe Pay Later Installment*
    - *PAYONE Paysafe Pay Later Invoice*
    - *Sofortüberweisung*
    - *PAYONE Paysafe Pay Later Debit*
    - *PAYONE iDeal*
    - *PAYONE EPS*
    - *PAYONE Secure Invoice*
    - *PAYONE Invoice*
    - *PAYONE Paydirekt*
    - *PAYONE Bancontact*
    - *PAYONE Prepayment*
    - *PAYONE Trustly*
    - *Direct debit*

>     - *Paydirekt*

Author: Woher hast du diese Zahlungsmethoden? Aus dem Video? Ich glaube nicht, dass die alle immer verfügbar sind, gerade, weil auch ratepay darunter ist. Ggf. als Frage aufnehmen, ob es vorinstallierte Zahlungsmethoden in Shopware 6 gibt oder ob man sich die immer individuell zusammenklicken kann. Alternativ würde ich einfach sowas schreiben wie: "All payment methods configured in Shopware 6 are displayed".

5. Click the *Tax classes* menu entry in the left side bar.  
  The tax classes available in Shopware 6 are displayed on the right side.  

> 6. Click the *Select Core1 tax class for Shopware 6 "Shopware tax class name"* drop-down list and select the applicable Core1 tax class. All available tax classes are displayed in the list. Repeat this step for all Shopware 6 tax classes.

Author: ```suggestion
6. Click the *Select Core1 tax class for Shopware 6 "Shopware tax class name"* drop-down list and select the corresponding tax class in the *Actindo Core1 Platform*. All available tax classes are displayed in the list. Repeat this step for all Shopware 6 tax classes.
```
Müssen grundsätzlich mal klären, ob wir von unserem System als "Actindo Core1 Platform" oder als "Actindo" oder wie auch immer sprechen...

7. If available, click the *Ratepay* menu entry in the left side bar.  
  The Ratepay settings are displayed on the right side.

  > [Info] The *Ratepay* menu entry is only displayed if the *Ratepay Integration for Shopware 6* plugin has been installed in your *Actindo Core1 Platform* and the *Ratepay Payment Plugin for Shopware 6* has been installed in your Shopware 6 account.
  
8. If desired, enable the *Enable Ratepay integration for this connection* toggle.  

>   > [Info] The Shopware 6 Ratepay extension provides additional information to the orders, which can be imported for further processing.  

Author: ```suggestion
  > [Info] The *Shopware 6 Ratepay extension* provides additional information to the orders, which can be imported for further processing.  
```

>   [comment]: <> (nicht standard / Extension für Treiber - beschreiben oder wegglassen?)

Author: Rückfrage, wie häufig die Extension genutzt wird.

9. Click the *Price rules* menu entry in the left side bar.  
  The price rule settings are displayed on the right side.

10. Enable the toggles for all price rules for which you want to create a price attribute in the *Actindo Core1 Platform* to maintain deviating prices. All price rules available in Shopware 6 are displayed. 

>   > [Info] You have to save the changes and synchronize the connection to create the corresponding price attribute. Once a price attribute has been created, the corresponding toggle is locked and cannot be disabled anymore.

Author: + Verweis auf synchronize a connection -> vgl. auch amazon
